src/signals/incident/components/form/ContainerSelect/ContainerLayer/ContainerLayer.tsx

```diff
@@ -32,49 +34,86 @@ export const ContainerLayer: FunctionComponent<DataLayerProps> = ({ featureTypes
   );
 
   const getFeatureType = useCallback(
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    feature => featureTypes.find(({ typeField, typeValue }) => feature.properties[typeField] === typeValue),
+    (feature: Feature) => featureTypes.find(({ typeField, typeValue }) => feature.properties[typeField] === typeValue),
     [featureTypes]
   );
 
-  const options: L.GeoJSONOptions<Record<string, string | undefined>> = useMemo(
+  type GeoJSONOptions = {
+    pointToLayer: (feature: Feature, latlng: LatLng) => L.Layer;
+  };
+  const options = useMemo<GeoJSONOptions>(
     () => ({
       pointToLayer: (feature, latlng: LatLng) => {
         const featureType = getFeatureType(feature);
         if (!featureType) return L.marker({ ...latlng, lat: 0, lng: 0 });
+        const selected = selection.some(
```

Comment:
After panning the map I get an error: 'TypeError: selection.some is not a function'